meson.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: meson
+  version: 0.63.2
+  epoch: 0
+  description: Fast and user friendly build system
+  target-architecture:
+    - noarch
```

Comment:
Wolfi doesn't have `noarch`.

Reply:
Ok great - should I change this to `all`?

Reply:
Yes!